docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -41,3 +41,22 @@ static class C
     public static void R<T>(ReadOnlySpan<T> s) => Console.Write(3);
 }
 ```
+
+When using C# 14 or newer and targeting .NET Framework or a .NET older than `net10.0`,
```

Comment:
Is it worth mentioning that, on .NET Framework, you need to be referencing `System.Memory`?